src/main/resources/application.yaml

```diff
@@ -1164,14 +1164,15 @@ job.scheduler:
           URL: ${spring.datasource.url}
           user: ${spring.datasource.username}
           password: ${spring.datasource.password}
+          maxConnections: 20
       jobStore:
         class: org.quartz.impl.jdbcjobstore.JobStoreTX
         isClustered: true
         driverDelegateClass: org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
         dataSource: jobscheduler
       threadPool:
         class: org.quartz.simpl.SimpleThreadPool
-        threadCount: 8
+        threadCount: 16
```

Comment:
Is there sufficient RAM and IO for the underlying Pods to support this?

Reply:
Good question, well not sure, but I think it should be ok

Reply:
At the moment we are scheduling the evidence reminders at midnight on same time. its temporary change, we are going to scale down once the fix we have provided for it